deploy/README.md

```diff
@@ -0,0 +1,33 @@
+# Deploying gpuctl
```

Comment:
> That way, it's easier for end-users to find.

That's kinda the point. End users don't need to know how to do all the frontend stuff, so that can be it's own file.

> and for us to keep updated

I'm not sure I follow this at all. Are you suggesting that my moving the deployment/fronend-dev docs into README.md, it somehow becomes easier to update.

---

That said, I'm sympathetic to wanting to have the end user docs at the front and center. I'll rejig them.

Reply:
> > and for us to keep updated
> 
> I'm not sure I follow this at all. Are you suggesting that my moving the deployment/fronend-dev docs into README.md, it somehow becomes easier to update.

Kinda, easier to keep up to date, not because it's less work but because we're more likely to notice old bits that need to be updated/removed. Case in point, the "main" README.md in the root directory of the repo is out of data with the changes made in this commit, as it gives a docker compose command, but makes no mention of it having to be run whilst in the `deploy/` directory.